src/Latestdocument.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {useState, useEffect} from 'react'
```

Comment:
Change the name of the file from lastestdocument to index.js using the format camel case for the names. 
And put the component in the directory components/LaststDocument/index.js